company/handbook/meetings.mdx

```diff
@@ -10,6 +10,10 @@ Being a remote-first company, we try to be as async as possible. When an issue a
 
 In other words, we have almost no (recurring) meetings and prefer written communication or quick Slack huddles.
 
+## Daily Standup
+
+Towards the end of each day, everyone on the Engineering and GTM teams should document their progress in the respective Slack standup channels, ensuring the team stays informed of important updates.
```

Comment:
On the eningeering side if they are working on something that takes longer than 1-2 days, add a ECD (estimated completion date) in stand up for that given item on when it will be pushed to prod